geometry_msgs/Pose gripper_pose
---
geometry_msgs/Pose leg_pose